### PR TITLE
*: support store statistic heartbeat.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @disksing @BusyJay @qiuyesuifeng

### CODE REVIEW

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -365,6 +374,7 @@ fn main() {
                 "store",
                 "set the path to rocksdb directory",
                 "/tmp/tikv/store");
+    opts.optopt("", "capacity", "set the store capacity", "1099511627776");
```
Comment (BusyJay) [inline src/bin/tikv-server.rs]: I think default should be unlimited or the disk's capacity.

**Comment by BusyJay** [comment]
LGTM

**Comment by disksing** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL

**Comment by BusyJay** [comment]
LGTM

**Comment by disksing** [comment]
LGTM

**Pull request merged**